### ISSUE TITLE
What should I use for cross-platform routing with `react-native-web`?

### ISSUE THREAD

**Issue by English3000** [necolas/react-native-web#927]
`react-router-native` is an ES module (ReactTraining/react-router#6115), meaning it is incompatible with browsers when compiled for my web app (which leads to a blank screen and a `SyntaxError` in the console).

Alternatively, will `react-router-dom` paired with `react-native-web` work for mobile devices?

**Comment by necolas** [comment]
Yeah you can use any DOM router

**Comment by vladp** [comment]
I am incorporating react-router (not react-router-native). Works fine with RNW.

To allow the button-click navigations to initiate re-renders of my RNW components, I incorporated pub/sub library https://www.npmjs.com/package/pubsub-js.

for my RN side of the app I used, now depreciated React Native Navigator, I will be replacing it with react-router-native.  

But at the end, it seems like I will end up with 2 different navigators one to work on web with RNW, and and the other to work with RN.  I hope (although I have not done RN side), that I can use React-router and react-router-native respectively.
